MVVM-C Modal Scene.xctemplate/___VARIABLE_sceneIdentifier___ViewModel.swift

```diff
@@ -7,6 +7,13 @@
 //
 
 import Foundation
+import FuntastyKit
+
+protocol ___VARIABLE_sceneIdentifier___CoordinatorInput: class, Coordinator {
```

Comment:
Jaký má ta `class` conformance důvod? Není lepší, pokud se to tady opakuje mít přímo `Coordinator` ve FuntastyKitu „dědící“ z `class`?

Reply:
Udělal jsem chybu, není potřeba vůbec! Coordinátor klidně nemusí být třída, i když by to byla docela blbost.

Reply:
Mi právě příjde, že je to poměrně v pořádku, například důvod proč by měl být `URLCoordinator` třída je poměrně nejasný.